src/pages/how-to/add-users-to-your-network.mdx

```diff
@@ -47,6 +47,25 @@ After logging in to the system, they will join your network automatically.
     This is a limitation that is likely to be removed in future versions.
 </Note>
 
+## Manage user roles
+NetBird has three user roles - `Owner`, `Admin`, and `User`. The roles allow you to control the level of access to the management API of your account.
+
+- `Owner` role - has full access to the account and can manage all aspects of the account. This role can not be assigned to multiple users.
```

Comment:
Mention account deletion and add an account deleteion doc reference.  
`This role can not be assigned to multiple users.`
There can be only one account owner in NetBird.